lib/postgres.go

```diff
@@ -391,6 +392,19 @@ type PGMetadataDerivedKey struct {
 	AccessSignature  []byte                           `pg:",type:bytea"`
 }
 
+// PGMetadataDAOCoinLimitOrder represents DAOCoinLimitOrderMetadata
+type PGMetadataDAOCoinLimitOrder struct {
+	tableName struct{} `pg:"pg_metadata_dao_coin_limit_order"`
+
+	TransactionHash            *BlockHash `pg:",pk,type:bytea"`
+	DenominatedCoinType        uint32     `pg:",use_zero"`
+	DenominatedCoinCreatorPKID *PKID      `pg:",type:bytea"`
+	DAOCoinCreatorPKID         *PKID      `pg:",type:bytea"`
+	OperationType              uint32     `pg:",use_zero"`
+	PriceNanos                 uint64     `pg:",use_zero"`
+	Quantity                   uint64     `pg:",use_zero"`
```

Comment:
Use a string to represent uint256's. See `PGMetadataDAOCoinTransfer` and how it uses `FromHex` to convert a hex string to a Uin2565 when reading from the database. and uses `uint256.Hex()` when creating a `PGMetadataDAOCoinTransfer` struct